app/src/main/java/com/epfl/drawyourpath/community/TournamentCreationFragment.kt

```diff
@@ -54,10 +89,99 @@ class TournamentCreationFragment : Fragment(R.layout.fragment_tournament_creatio
     private fun createCreateButton(view: View) {
         val createButton = view.findViewById<Button>(R.id.tournament_creation_create_button)
         createButton.setOnClickListener {

```

Comment:
Yes it would be much cleaner this way, I'm changing this. Thank you for pointing it out. I tried not touching too much of what was already done but some part of the logic and functions naming are debatable to say the least...